sky/packages/sky/lib/src/rendering/statistics_box.dart

```diff
@@ -0,0 +1,47 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:sky/src/rendering/box.dart';
+import 'package:sky/src/rendering/object.dart';
+
+class StatisticsBox extends RenderBox {
+
+  StatisticsBox({int optionsMask: 0}) : _optionsMask = optionsMask;
+
+  int _optionsMask;
+  int get optionsMask => _optionsMask;
```

Comment:
put the getter above the field, so that when we add dartdocs the doc is before the entire paragraph, not in the middle